src/modules/endpoint/reducers/morphContentLanguage.js

```diff
@@ -40,11 +40,13 @@ const categoryRouteTranslator = (newCategoriesMap: CategoriesMapModel, newLangua
       children: resultChildren,
       depth,
       allAvailableLanguages,
-      language: newLanguage
+      language: newLanguage,
+      city,
+      peek
     }
   }
 
-const eventRouteTranslator = (newEvents: Array<EventModel>, newLanguage: string) =>
+const eventRouteTranslator = (newEvents: Array<EventModel>, city: string, newLanguage: string) =>
```

Comment:
Right now city isn't used

Reply:
Removed, weird that there was no warning